crates/transaction-pool/tests/it/best.rs

```diff
@@ -0,0 +1,10 @@
+//! Best transaction and filter testing
+
+use reth_transaction_pool::{noop::NoopTransactionPool, BestTransactions, TransactionPool};
+
+#[test]
+fn test_best_transactions() {
+    let noop = NoopTransactionPool::default();
+    let mut best = noop.best_transactions().filter_transactions(|_| true);
+    assert!(best.next().is_none());
+}
```

Comment:
let's tracck a better test, this does nothing rn :smile:

Reply:
yeah this just ensures that I can call this on `Box< dyn>`